pages/categories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import Button from "@material-ui/core/Button";
+
+function Categories() {
+  return (
+    <div>
+      <Button>Hello</Button>
```

Comment:
This goes for all of them, but maybe do a text field instead with the name of the page, so that we can actually test that the different pages work as intended.